test/unit/ObjectMakrTests/proliferate.js

```diff
@@ -0,0 +1,51 @@
+define(["mocks"], function (mocks) {
```

Comment:
Protip: there are already tests for `proliferate` under EightBittr. You can just copy & paste those.